test/cases/testdata/jsonbuiltins/test-json-marshal-indent.yaml

```diff
@@ -0,0 +1,36 @@
+---
+cases:
+  - data: {}
+    modules:
+      - |
+        package test
+
+        p = x {
+          json.marshal_indent([1234500000 + 67890, 1e6 * 2, 1e109 / 1e100], "  ", x)
+        }
+    note: jsonbuiltins/marshal_indent-explicit
+    query: data.test.p = x
+    want_result:
+      - x: |-
+          [
+            1234567890,
+            2000000,
+            1000000000
+          ]
+  - data: {}
+    modules:
+      - |
+        package test
+
+        p = x {
+          json.marshal_indent([1234500000 + 67890, 1e6 * 2, 1e109 / 1e100], null, x)
```

Comment:
It'd be great to have both an array and object test case for this new `json.marshal_indent` built-in.

Reply:
Can do! Let me work on that.

Reply:
Added new tests in aad355a99547f4679672f2208e82c22529616204. Let me know if there's any additional tests you'd like to see here.